test/unit/middleware/authentication.js

```diff
@@ -0,0 +1,35 @@
+import sinon from 'sinon';
+import chai from 'chai';
+import authenticate from '../../../middleware/authenticate';
+
+const should = chai.should();
+
+describe('middlewares', () => {
+  context('authenticate()', () => {
```

Comment:
#### Nitpick
- Yep, please use english sentences to describe your tests throughout.

Reply:
Noted. Will make modification